src/ol/render/replay.js

```diff
@@ -22,16 +21,16 @@ _ol_render_replay_.ORDER = [
  * @const
  * @enum {number}
  */
-_ol_render_replay_.TEXT_ALIGN = {};
-_ol_render_replay_.TEXT_ALIGN['left'] = 0;
-_ol_render_replay_.TEXT_ALIGN['end'] = 0;
-_ol_render_replay_.TEXT_ALIGN['center'] = 0.5;
-_ol_render_replay_.TEXT_ALIGN['right'] = 1;
-_ol_render_replay_.TEXT_ALIGN['start'] = 1;
-_ol_render_replay_.TEXT_ALIGN['top'] = 0;
-_ol_render_replay_.TEXT_ALIGN['middle'] = 0.5;
-_ol_render_replay_.TEXT_ALIGN['hanging'] = 0.2;
-_ol_render_replay_.TEXT_ALIGN['alphabetic'] = 0.8;
-_ol_render_replay_.TEXT_ALIGN['ideographic'] = 0.8;
-_ol_render_replay_.TEXT_ALIGN['bottom'] = 1;
-export default _ol_render_replay_;
+export const TEXT_ALIGN = {
+  left: 0,
+  end: 0,
+  center: 0.5,
+  right: 1,
+  start: 1,
+  top: 0,
+  middle: 0.5,
+  hanging: 0.2,
+  alphabetic: 0.8,
+  ideographic: 0.8,
+  bottom: 1
```

Comment:
@raiyni - since these properties are accessed with user-provided strings, we need to quote them to avoid compiler renaming.  The safest thing would be to use bracket notation - quoted properties in an object literal may be unquoted by a babel transform before making it to the compiler.